src/sidebar.js

```diff
@@ -0,0 +1,328 @@
+    }
+
+    if (ext === 'isml') {
+      return `Template:&nbsp; <span>${title}</span>`
+    }
+
+    return `Title:&nbsp; <span>${title}</span>`
+  }
+
+  /**
+   * Get Operating System ( for native font support )
+   */
+  const getOS = () => {
+    const platform = window.navigator.platform
+    const mac = ['Macintosh', 'MacIntel', 'MacPPC', 'Mac68K']
+    const windows = ['Win32', 'Win64', 'Windows', 'WinCE']
+
+    if (mac.indexOf(platform) !== -1) {
+      return 'platform-mac'
+    } else if (windows.indexOf(platform) !== -1) {
+      return 'platform-windows'
+    } else if (/Linux/.test(platform)) {
+      return 'platform-linux'
+    }
+  }
+
+  /**
+   * Get Property from Comment
+   * @param {string} comment
+   * @param {string} prop
+   */
+  const getProp = (comment, prop) => {
+    var regex = new RegExp(prop + '="([^"]+)"')
+    var match = comment.match(regex)
+
+    return (match) ? match[1] : null
+  }
+
+  /**
+   * Inspect Selected DOM Element and check for Comment Blocks
+   */
+  const inspect = () => {
+    // We don't really care if there is an error here, as nothing would have loaded anyway, so we can skip it
+    const onError = () => undefined
+
+    // Get Selected Element
+    const selectedElement = (result, exception) => {
+      if (exception) {
+        return
+      }
+
+      // Some browsers return selected element as an array
+      if (Array.isArray(result)) {
+        result = result[0]
+      }
+
+      // Cache Sidebar DOM Selection
+      const $sidebar = document.getElementById('sidebar')
+
+      // Check if we have a result from our selection
+      if (result) {
+        // Before we use the result, get some details about the page we're on
+        const getPageInfo = location => {
+          if (location && typeof location[0] !== 'undefined') {
+            const url = new URL(location[0])
+            const domain = url.hostname
+            const match = url.pathname.match(/\/s\/([a-zA-Z0-9_-]+)\/*/)
+            const siteID = (match) ? match[1] : null
+
+            // Parse Comment & Update Side Bar
+            const info = parseCommentInfo(result)
+
+            if (info) {
+              $sidebar.innerHTML = generateSidebar(info, domain, siteID)
+              bindEvents()
+            } else {
+              $sidebar.innerHTML = '<i>No SFCC Comment Selected</i>'
+            }
+          }
+        }
+
+        // Get Current Page URL Info
+        if (browser.devtools) {
+          browser.devtools.inspectedWindow.eval(`{ window.location.href }`).then(getPageInfo, onError)
+        }
+      } else {
+        $sidebar.innerHTML = '<i>No SFCC Comment Selected</i>'
+      }
+    }
+
+    // Use inspector to parse DOM and return node if Comment Block ( nodeType === 8 )
+    if (browser.devtools) {
+      browser.devtools.inspectedWindow.eval(`{ const node = $0; node.nodeType === 8 && node.nodeValue; }`).then(selectedElement, onError)
```

Comment:
The node type constant may be easier to understand
```suggestion
      browser.devtools.inspectedWindow.eval(`{ const node = $0; node.nodeType === Node.COMMENT_NODE && node.nodeValue; }`).then(selectedElement, onError)
```